etsin_finder/frontend/js/components/qvain/general/selectedItems.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { inject, observer } from 'mobx-react'
+import Label from './label'
+
+class SelectedItems extends Component {
+  generateComponents = () => {
+    const { getter } = this.props
+    const { lang } = this.props.Stores.Locale
+
+    return getter.map((item) => (
+      <Label color="#007fad" margin="0 0.5em 0.5em 0" key={`selected_${item.url}`}>
```

Comment:
It's better to use color names from the theme when possible. `#007fad` is `primary` in the theme.
```suggestion
      <Label color="primary" margin="0 0.5em 0.5em 0" key={`selected_${item.url}`}>
```
Not a big deal since hardcoded colors are currently used in a lot of places in the code anyways. 

In a related matter: When creating styled components, you can use colors (or other values) from theme.js in CSS template strings like this:
```css
  border-bottom: 1px solid ${props => props.theme.color.primary};
```
This is due to the use of a theme provider `<ThemeProvider theme={etsinTheme}>`.